doc/compodoc_sources/concepts/configuration.md

```diff
@@ -362,6 +364,28 @@ Example:
 
 ```
 
+#### Permissions
+Permissions for interaction on entities can be given or denied using the config.
+This will disable buttons in the app to create, delete or edit entities if the user is not permitted.
+
+The following example will only allow `admin` users to create, edit and delete `School` objects:
+```
+"entity:School": {
+  "permissions": {
+    "create": ["admin"],
+    "update": ["admin"],
+    "delete": ["admin"]
+  }
+}
+```
+Buttons can be marked as part of an interaction using the `appEntityInteraction` directive:
+```
+<button
+  (click)="switchEdit()"
+  [appEntityOperation]="{entity: entity?.getConstructor(), operation: operationType.UPDATE}"
```

Comment:
The name `appEntityOperation` doesn't indicate in any way that this relates to disabling something or that it has to do with permissions. Maybe it could be called `appRestrictToEntityOperationPermission` or another way more explicit way?

Reply:
I chose this name because I did not want to make it dependent on what the directive actually does. This directive should only _mark_ the buttons which belong to entity operations. 
At a later point this should allow us, for example, to log every time a `CREATE`, `DELETE` or `UPDATE` operation is performed on a entity.